pkg/types/interfacetests/chain_reader_interface_tests.go

```diff
@@ -242,8 +242,9 @@ func runQueryKeyInterfaceTests[T TestingT[T]](t T, tester ChainReaderInterfaceTe
 
 				ts := &TestStruct{}
 				assert.Eventually(t, func() bool {
+					// sequences from queryKey without limit and sort should be in descending order
 					sequences, err := cr.QueryKey(ctx, AnyContractName, query.KeyFilter{Key: EventName}, query.LimitAndSort{}, ts)
-					return err == nil && len(sequences) == 2 && reflect.DeepEqual(&ts1, sequences[0].Data) && reflect.DeepEqual(&ts2, sequences[1].Data)
+					return err == nil && len(sequences) == 2 && reflect.DeepEqual(&ts2, sequences[0].Data) && reflect.DeepEqual(&ts1, sequences[1].Data)
```

Comment:
How did these end up switched?

Reply:
It wasn't really switched because the order was random. I made changes in core so that the orders is always descending